Dockerfile

```diff
@@ -2,9 +2,10 @@ ARG quay_expiration=never
 ARG release_tag=0.0.0
```

Comment:
You can add the arg here with the default (=amd64). Then you don't even have to change build-main.yaml.